journalpump/senders/kafka.py

```diff
@@ -110,9 +110,14 @@ def send_messages(self, *, messages, cursor):
         if not self.kafka_producer:
             self._init_kafka()
         try:
-            for msg in messages:
-                self.kafka_producer.send(topic=self.topic, value=msg, key=self.kafka_msg_key)
+            list_fut_record_metadata = [
```

Comment:
small: `list_fut_record_metadata` not a super descriptive name. Could it be `future_results` or smth. what this list contaons?

Reply:
also this part: `... for msg in messages` kinda hints that they are messages...

Reply:
List of futures that resolve to record metadata objects.. but yep, could use result_futures as we do not actually care about the success values.